.github/ISSUE_TEMPLATE/bug-report--hash-engine-.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report (HASH Engine)
+about: Create a report to help us improve HASH Engine
+title: "[BUG]"
+labels: A-engine, C-bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Where possible, please link us to your simulation project on HASH to help us debug. Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```

Comment:
Probably want to replace this with a request for similar information as `error-stack`. Perhaps adding a request for the Python version.